tiled/server/app.py

```diff
@@ -409,6 +409,8 @@ def override_get_settings():
         ]:
             if authentication.get(item) is not None:
                 setattr(settings, item, authentication[item])
+            if authentication.get("single_user_api_key") is not None:
+                settings.single_user_api_key_generated = False
```

Comment:
Move this out of the for-loop above?

```suggestion
        if authentication.get("single_user_api_key") is not None:
            settings.single_user_api_key_generated = False
```